### ISSUE TITLE
SQLite error: Dino crashes on launch, Linux Mint 19.2

### ISSUE THREAD

**Issue by rscmbbng** [dino/dino#982]
On launch nothing happens and when launched from terminal the following error is displayed:

```
** (dino:10923): ERROR **: 11:40:53.789: database.vala:142: SQLite error: 1 - near "ON": syntax error: INSERT INTO settings (key, value) VALUES (?, ?) ON CONFLICT (key) DO UPDATE SET value=excluded.value
Trace/breakpoint trap (core dumped) 
```

This also happens after reinstallation and with an empty ~/.local/share/dino

OS is Linux Mint 19.2, 32bit

```
uname -a 
Linux interstellar 4.15.0-128-generic #131-Ubuntu SMP Wed Dec 9 06:53:22 UTC 2020 i686 i686 i686 GNU/Linux 
```

Installed via OBS repo, version 0.2.0~git27.20210121.f12f3c371-1

Used to work fine before!

**Comment by fiaxh** [comment]
Linux Mint 19.2 builds on Ubuntu 18.04 and contains sqlite 3.22, however Dino requires 3.24 since 0beb592c5a6a2767e11a892bdb5ac9bcc5283c38. The SQL statement referred to in your error is only available above 3.24.

Dino shouldn't even build/install on your system - how did you get it to build/install?

**Comment by rscmbbng** [comment]
Hey thanks for the swift reply. It seems like I'll have to upgrade the distro, as newer versions of linux mint no longer support 32bit. 

Dino was installed via [OBS](https://software.opensuse.org/download.html?project=network:messaging:xmpp:dino&package=dino) which lists Ubuntu 18.04 and 17.04 as compatible. Perhaps that is something to be changed if compatibility with those distros  is no longer the case?
